examples/gallery/symbols/patterns.py

```diff
@@ -0,0 +1,76 @@
+r"""
+Bit and Hachure Patterns
+------------------------
+
+PyGMT allows using bit or hachure patterns via the ``fill`` parameter
+or similar parameters:
+
+- Symbols and polygons: :meth:`pygmt.Figure.plot` via ``fill``
+- Histogram bars or sectors: :meth:`pygmt.Figure.histogram` or
+  :meth:`pygmt.Figure.rose` via ``fill``
+- Anomalies: :meth:`pygmt.Figure.wiggle` via ``fillpositive``
+  and ``fillnegative``
+- Land and water masses: :meth:`pygmt.Figure.coast` via ``land``
+  and ``water``
+- Focal mechanisms: :meth:`pygmt.Figure.meca` via ``G`` and ``E``
+
+The required argument has the following form:
+
+**P**\ |**p**\ *pattern*\ [**+b**\ *color*\ ][**+f**\ *color*\ ][**+r**\ *dpi*]
+
+
+TODO
+
+An overview of all 90 predefined bit and hachure patterns can by found at
+https://docs.generic-mapping-tools.org/latest/cookbook/predefined-patterns.html.
+"""
+
+import pygmt
+
+y = 13
+
+fig = pygmt.Figure()
+fig.basemap(
+    region=[0, 10, 0, 12],
+    projection="X10c",
+    frame="rlbt+glightgray+tBit and Hachure Patterns",
+)
+
+# To use a pattern as fill append "p" and the number of the desired
+# pattern. By default, the pattern is plotted in black and white
```

Comment:
```suggestion
# patterns. By default, the pattern is plotted in black and white
```

Reply:
Hm. Currently I am not sure about changing this to plural.

Reply:
You're right, just ignore the change :wink:

Reply:
No worries @michaelgrund!